arrows/klv/klv_checksum.cxx

```diff
@@ -0,0 +1,90 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+/// \file
+/// Implementation of the KLV checksum functions.
+
+#include "klv_checksum.h"
+
+#include <numeric>
+#include <vector>
+
+#include <cstdlib>
```

Comment:
BTW, what needs `cstdlib`? If it's for `size_t`, would `cstdint` + `cstddef` be better?